classes/item.rb

```diff
@@ -2,7 +2,7 @@
 
 class Item
```

Comment:
Let's create a method called `add_label`, so it can be used when you add an item with `add_item` method in `Label` class, we will used the same way for an `Author` and `Genre` as well, and we will have `add_genre` and `add_author` methods.